website/docs/plugin/framework/migrating/attributes-blocks/default-values.mdx

```diff
@@ -7,13 +7,13 @@ description: >-
 
 # Default Values
 
-Default values set a value for an attribute when the Terraform configuration does not provide one. In SDKv2, default
-values are set via fields on an attribute's schema. In the Framework, you set default values via plan modification.
+Default values set a value for an attribute when the Terraform configuration does not provide one. In SDKv2 and the
+Framework default values are set via the `Default` field on an attribute's schema.
```

Comment:
I think it might be good to state upfront here that default value support is only available in the framework for resources and what to do for provider and data source schemas.

The description above needs a tweak as well 👍